journal/original/24-chapter-21.adoc

```diff
@@ -372,7 +372,7 @@ on behalf of the assembly and himself, their thankfulness and satisfaction,
 and how welcome my visit had been to them; because, he said,
 '`You have preached to us the everlasting gospel,
 and have shown to us the propitiatory sacrifice for the sins of mankind,
-Christ Jesus with something further, which when interpreted,
+Christ Jesus.`" with something further, which when interpreted,
```

Comment:
Because the quote (which starts on line 373) starts with a _single_ quote, this close quote should match, or we should change the other one to a double. Probably the latter I guess.